doc/source/docker/run-quickstart-examples-docker-compose.rst

```diff
@@ -39,13 +39,16 @@ Run the Quickstart Example
    .. code-block:: bash
        :substitutions:
 
-       $ curl https://raw.githubusercontent.com/adap/flower/refs/tags/v|stable_flwr_version|/src/docker/complete/compose.yml \
+       $ curl https://raw.githubusercontent.com/adap/flower/refs/heads/main/src/docker/complete/compose.yml \
```

Comment:
Can we update the link to the commit where we set the version to 1.12.0? I noticed that if we use main, the files won’t work once I merge the Docker Compose changes for 1.13.0.
```suggestion
       $ curl https://raw.githubusercontent.com/adap/flower/24b2861465431a5ab234a8c4f76faea7a742b1fd/src/docker/complete/compose.yml \
```